fastlane/Fastfile

```diff
@@ -1152,6 +1150,23 @@ platform :ios do
     end
   end
 
+  desc "Clones or updates paywall-preview-resources repo"
+  lane :fetch_paywall_preview_resources do
+    Dir.chdir("../Tests") do
+      if !File.directory?(paywall_preview_resources_repo_name)
+        UI.important("Cloning paywall-preview-resources repo")
+        sh "git", "clone", "git@github.com:RevenueCat/#{paywall_preview_resources_repo_name}.git"
+      end
```

Comment:
I wonder if we can do a shallow clone? The thing is that the commit is not necessarily HEAD right?

Reply:
Yeah, this is a good point. But I don't know if we should be making many assumptions about this commit. It could potentially lead to future bugs. But on the other hand, doing a shallow clone on the resources repo seems like a proper thing to do. 

> the commit is not necessarily HEAD right?

The's correct, I think